apps/src/StudioApp.js

```diff
@@ -2279,7 +2279,8 @@ StudioApp.prototype.handleEditCode_ = function(config) {
   }
 
   // Remove maker API blocks from palette, unless maker APIs are enabled.
-  if (!project.getMakerAPIs()) {
+  // Also check config.level to ensure maker toolkit shows when editing start mode
+  if (!project.getMakerAPIs() && !config.level.makerlabEnabled) {
```

Comment:
what happens in the following situation? is it even possible?
1. a level has maker enabled
2. the user disables maker from the "settings" cog
3. reload the page

my assumption would be that `project.getMakerAPIs()` returns falsy, but `config.level.makerlabEnabled` is still true, which means the maker blocks would still display in the toolbox, which i think is unexpected

Reply:
hmm great point! I'll look into it

Reply:
Added logic to make check if the level is specifically `inStartMode`! Also updated the comment above, let me know if it makes sense:

https://github.com/code-dot-org/code-dot-org/pull/44156/commits/aa2de090f614a7e7d983a7d8d7373f162f641fc5